### PR TITLE
Update layout for workflow events, actions

### PR DESCRIPTION
**Pull request by tanhongit**

### CODE REVIEW

File: resources/views/events/github/push/default.blade.php [hunk 1]
```diff
@@ -30,4 +30,4 @@
 ) !!}
 @endforeach
 
-{!! __('tg-notifier::events/github/push.default.pusher', ['name' => $payload->pusher->name]) !!}
+👤 {!! __('tg-notifier::events/github/push.default.pusher') !!} : <code>{{ $payload->pusher->name }}</code>
```
Comment (cambustt30) [inline resources/views/events/github/push/default.blade.php:33]:
```suggestion
👤 {!! __('tg-notifier::events/github/push.default.pusher') !!}: <code>{{ $payload->pusher->name }}</code>
```

Comment (tanhongit) [reply resources/views/events/github/push/default.blade.php]: @cambustt30 Nice, thank you!

**Review by tanhongit — COMMENTED** [review]
@cslant/souvenir-tcom  These are events that have not been updated:
- workflow_job 
- pull_request
- label
- deploy_key

File: resources/views/events/github/branch_protection_rule/deleted.blade.php [hunk 2]
```diff
@@ -6,9 +6,9 @@
...
 
-{!! __('tg-notifier::events/github/branch_protection_rule.deleted.title', [
-    'repo' => "<a href='$repository->html_url'>$repository->full_name</a>"
+🗑 {!! __('tg-notifier::events/github/branch_protection_rule.deleted.title', [
+    'repo' => "🦑<a href='$repository->html_url'>$repository->full_name</a>"
         ]
     ) !!}
 
-📢 <b>{{ $payload->rule->name }}</b>
+🛡 {!! __('tg-notifier::events/github/branch_protection_rule.name') !!}: <b>{{ $payload->rule->name }}</b>
```
Comment (cambustt30) [inline resources/views/events/github/branch_protection_rule/deleted.blade.php:14]:
```suggestion
🛡 {!! __('tg-notifier::events/github/branch_protection_rule.name') !!}: <code>{{ $payload->rule->name }}</code>
```

File: resources/views/events/github/branch_protection_rule/edited.blade.php [hunk 3]
```diff
@@ -6,19 +6,19 @@
...
 
 @if(isset($payload->changes->name->from))
-{!! __('tg-notifier::events/github/issues.edited.changes.title.from', ['title_from' => $payload->changes->name->from]) !!}
-{!! __('tg-notifier::events/github/issues.edited.changes.title.to', ['title_to' => $payload->rule->name]) !!}
+📖 {!! __('tg-notifier::events/github/branch_protection_rule.edited.changes.title.name') !!}
+    📝 {!! __('tg-notifier::events/github/issues.edited.changes.title.from', ['title_from' => $payload->changes->name->from]) !!}
+    🛡 {!! __('tg-notifier::events/github/issues.edited.changes.title.to', ['title_to' => $payload->rule->name]) !!}
 @else
-📢 <b>{{ $payload->rule->name }}</b>
+🛡 {!! __('tg-notifier::events/github/branch_protection_rule.name') !!}: <b>{{ $payload->rule->name }}</b>
```
Comment (cambustt30) [inline resources/views/events/github/branch_protection_rule/edited.blade.php:19]:
```suggestion
🛡 {!! __('tg-notifier::events/github/branch_protection_rule.name') !!}: <code>{{ $payload->rule->name }}</code>
```

File: resources/views/events/github/issues/closed.blade.php [hunk 4]
```diff
@@ -7,13 +7,12 @@
...
-{!! __('tg-notifier::events/github/issues.closed.title', [
-            'issue' => "<a href='$issue->html_url'>{$payload->repository->full_name}#$issue->number</a>",
+🚫 {!! __('tg-notifier::events/github/issues.closed.title', [
+            'issue' => "🦑 <a href='$issue->html_url'>{$payload->repository->full_name}#$issue->number</a>",
             'user' => "<a href='{$payload->sender->html_url}'>@{$payload->sender->login}</a>"
         ]
     ) !!}
 
-📢 <b>{{ $issue->title }}</b>
+📢 {!! __('tg-notifier::events/github/issues.issue_title') !!}: <b>{{ $issue->title }}</b>
```
Comment (cambustt30) [inline resources/views/events/github/issues/closed.blade.php:16]:
```suggestion
📢 {!! __('tg-notifier::events/github/issues.issue_title') !!}: <code>{{ $issue->title }}</code>
```

File: resources/views/events/github/issues/deleted.blade.php [hunk 5]
```diff
@@ -7,13 +7,12 @@
...
-{!! __('tg-notifier::events/github/issues.deleted.title', [
-            'issue' => "<a href='$issue->html_url'>{$payload->repository->full_name}#$issue->number</a>",
+🗑 {!! __('tg-notifier::events/github/issues.deleted.title', [
+            'issue' => "🦑 <a href='$issue->html_url'>{$payload->repository->full_name}#$issue->number</a>",
             'user' => "<a href='{$payload->sender->html_url}'>@{$payload->sender->login}</a>"
         ]
     ) !!}
 
-📢 <b>{{ $issue->title }}</b>
+📢 {!! __('tg-notifier::events/github/issues.issue_title') !!}: <b>{{ $issue->title }}</b>
```
Comment (cambustt30) [inline resources/views/events/github/issues/deleted.blade.php:16]:
```suggestion
📢 {!! __('tg-notifier::events/github/issues.issue_title') !!}: <code>{{ $issue->title }}</code>
```

File: resources/views/events/github/issues/edited.blade.php [hunk 6]
```diff
@@ -7,20 +7,20 @@
...
-{!! __('tg-notifier::events/github/issues.edited.title', [
-            'issue' => "<a href='$issue->html_url'>{$payload->repository->full_name}#$issue->number</a>",
+⚠️ {!! __('tg-notifier::events/github/issues.edited.title', [
+            'issue' => "🦑 <a href='$issue->html_url'>{$payload->repository->full_name}#$issue->number</a>",
             'user' => "<a href='{$payload->sender->html_url}'>@{$payload->sender->login}</a>"
         ]
     ) !!}
 
-📢 <b>{{ $issue->title }}</b>
+📢 {!! __('tg-notifier::events/github/issues.issue_title') !!}: <b>{{ $issue->title }}</b>
```
Comment (cambustt30) [inline resources/views/events/github/issues/edited.blade.php:16]:
```suggestion
📢 {!! __('tg-notifier::events/github/issues.issue_title') !!}: <code>{{ $issue->title }}</code>
```

File: resources/views/events/github/issues/locked.blade.php [hunk 7]
```diff
@@ -7,13 +7,12 @@
...
-{!! __('tg-notifier::events/github/issues.locked.title', [
-            'issue' => "<a href='$issue->html_url'>{$payload->repository->full_name}#$issue->number</a>",
+🔒 {!! __('tg-notifier::events/github/issues.locked.title', [
+            'issue' => "🦑 <a href='$issue->html_url'>{$payload->repository->full_name}#$issue->number</a>",
             'user' => "<a href='{$payload->sender->html_url}'>@{$payload->sender->login}</a>"
         ]
     ) !!}
 
-📢 <b>{{ $issue->title }}</b>
+📢 {!! __('tg-notifier::events/github/issues.issue_title') !!}: <b>{{ $issue->title }}</b>
```
Comment (cambustt30) [inline resources/views/events/github/issues/locked.blade.php:16]:
```suggestion
📢 {!! __('tg-notifier::events/github/issues.issue_title') !!}: <code>{{ $issue->title }}</code>
```

File: resources/views/events/github/issues/closed.blade.php (hunk 4 above)
Comment (tanhongit) [reply resources/views/events/github/issues/closed.blade.php:16]: thank you for the suggestion!!
lgtm!

**Review by tanhongit — COMMENTED** [review]
@cslant/souvenir-tcom deploy_key have been updated

File: resources/views/events/github/branch_protection_rule/edited.blade.php [hunk 8]
```diff
@@ -6,19 +6,19 @@
...
+    🛡 {!! __('tg-notifier::events/github/issues.edited.changes.title.to', ['title_to' => $payload->rule->name]) !!}
 @else
-📢 <b>{{ $payload->rule->name }}</b>
+🛡 {!! __('tg-notifier::events/github/branch_protection_rule.name') !!}: <code>{{ $payload->rule->name }}</code>
 @endif
-
-{!! __('tg-notifier::events/github/branch_protection_rule.edited.link', [
-    'link' => "<a href='$repository->html_url/settings/branch_protection_rules/{$payload->rule->id}'>{$repository->html_url}/settings/branch_protection_rules/{$payload->rule->id}</a>"
+🔗 {!! __('tg-notifier::events/github/branch_protection_rule.edited.link', [
+    'link' => "<a href='$repository->html_url/settings/branch_protection_rules/{$payload->rule->id}'>{$payload->rule->id}</a>"
```
Comment (tnphtcom) [inline resources/views/events/github/branch_protection_rule/edited.blade.php:22]: We need to add '#' to synchronize with other events

```suggestion
    'link' => "<a href='$repository->html_url/settings/branch_protection_rules/{$payload->rule->id}'>#{$payload->rule->id}</a>"
```

File: resources/views/events/github/branch_protection_rule/created.blade.php [hunk 9]
```diff
@@ -6,14 +6,13 @@
...
         ]
     ) !!}
 
-📢 <b>{{ $payload->rule->name }}</b>
-
-{!! __('tg-notifier::events/github/branch_protection_rule.created.link', [
-    'link' => "<a href='$repository->html_url/settings/branch_protection_rules/{$payload->rule->id}'>{$repository->html_url}/settings/branch_protection_rules/{$payload->rule->id}</a>"
+🛡 {!! __('tg-notifier::events/github/branch_protection_rule.name') !!}: <code>{{ $payload->rule->name }}</code>
+🔗 {!! __('tg-notifier::events/github/branch_protection_rule.created.link', [
+    'link' => "<a href='$repository->html_url/settings/branch_protection_rules/{$payload->rule->id}'>{$payload->rule->id}</a>"
```
Comment (tnphtcom) [inline resources/views/events/github/branch_protection_rule/created.blade.php:16]:
```suggestion
    'link' => "<a href='$repository->html_url/settings/branch_protection_rules/{$payload->rule->id}'>#{$payload->rule->id}</a>"
```

Comment (tanhongit) [reply resources/views/events/github/branch_protection_rule/created.blade.php:16]: thanks for reminding!

File: lang/en/events/github/pull_request.php [hunk 10]
```diff
@@ -3,34 +3,34 @@
 return [
     'review' => '👥 Reviewers: ',
```
Comment (tanhongit) [inline lang/en/events/github/pull_request.php:4]: @tongthanhphu Please delete the redundant icon in the lang files

Comment (tongthanhphu) [reply lang/en/events/github/pull_request.php]: it has been fixed

**Review by tanhongit — COMMENTED** [review]
the #40 will be closed even though not all events are complete. Missing events will be split into small issues or projects.

**Pull request merged**